backstop/tests/slider.html

```diff
@@ -0,0 +1,349 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+
+<!DOCTYPE html>
+<html id="theme">
+
+  <head>
+    <title>Slider | iTwin UI</title>
+    <link
+      rel="icon"
+      href="../assets/favicon.ico"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <style>
+      section[id^='demo'] {
+        width: auto;
+      }
+
+      /* Gives left & right labels an equal width so the slider is centered. */
+      .iui-left,
+      .iui-right {width: 25px;}
```

Comment:
Which part of centering you mean?
Cause disabling it just makes slider full width and aligned with title:
![image](https://user-images.githubusercontent.com/81580355/124258253-dc421500-db35-11eb-8264-3ea3fdb4e451.png)

If something is still needed, would be good to tackle not direct classes, cause then its confusing which part is coming from styles, which part is hardcoded for demo purposes...

Reply:
If the left & right labels do not have a shared fixed width, the slider will become off center depending on how long the labels are.
![Screen Shot 2021-07-02 at 4 09 21 PM](https://user-images.githubusercontent.com/849817/124323792-e07b2c00-db4f-11eb-8692-d49ffe4af169.png)
![width-issue](https://user-images.githubusercontent.com/849817/124323631-a27e0800-db4f-11eb-9de7-c195d39c4c7f.png)

Reply:
well, not sure in our examples its critical.. When removing this rule, we only get a bit longer slider itself (labels are still aligned).
![image](https://user-images.githubusercontent.com/81580355/124917444-85887f80-dffc-11eb-90c8-eb5e3b9827f9.png)
Compared with fixed width:
![image](https://user-images.githubusercontent.com/81580355/124917501-976a2280-dffc-11eb-9602-08849bca65dd.png)

Reply:
Cause if we expect users to have it also aligned the way we have it, we will need to add it in react somehow (if not provided from css by default...).

Reply:
It is pretty minor and only becomes more obvious the wider the range is (1-1,000,000 is obvious where 100,000-1,000,000 is not).  We can choose to skip this for now.